### PR TITLE
Add RPM packaging.

### PR DESCRIPTION
**Pull request by Rudd-O**

### CODE REVIEW

File: seedvault-backup-parser.spec [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+%define mybuildnumber %{?build_number}%{?!build_number:1}
+
+Name:           seedvault-backup-parser
+Version:        2020.10.23
+Release:        %{mybuildnumber}%{?dist}
+Summary:        Parse Seedvault backups
+BuildArch:      noarch
+
+License:        GPLv2+
+URL:            https://github.com/Rudd-O/seedvault_backup_parser
```
Comment (tlambertz) [inline seedvault-backup-parser.spec:12]: This is not the link of this repository, but your fork

File: seedvault-backup-parser.spec [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+mkdir -p "$RPM_BUILD_ROOT"/%{_bindir}
+cp -f parse.py "$RPM_BUILD_ROOT"/%{_bindir}/%{name}
+chmod 755 "$RPM_BUILD_ROOT"/%{_bindir}/%{name}
+
+%files
+%attr(0755, root, root) %{_bindir}/%{name}
+%doc README.md LICENSE
+
+%changelog
+* Fri Oct 23 2020 Manuel Amador (Rudd-O) <rudd-o@rudd-o.com>
```
Comment (tlambertz) [inline seedvault-backup-parser.spec:43]: It seems wrong to just provide you as author here

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+// https://github.com/Rudd-O/shared-jenkins-libraries
+@Library('shared-jenkins-libraries@master') _
```
Comment (tlambertz) [inline Jenkinsfile:2]: what do these do? Does having jenkins give us any advantage?

**Review by tlambertz — CHANGES_REQUESTED** [review]
I feel it is a bit overkill to package a simple python script. What is your use-case for this?

File: seedvault-backup-parser.spec (hunk 2 above)
Comment (Rudd-O) [reply seedvault-backup-parser.spec:43]: It's normal to provide the contact info for the packager in the `%changelog` tag.  Try `rpm -q --changelog <package>` on any RPM-based distribution.  It's usually the person modifying the RPM specfile.

File: Jenkinsfile (hunk 3 above)
Comment (Rudd-O) [reply Jenkinsfile:2]: Actually, I don't have to add the `Jenkinsfile`.  I'll remove this from the commit.

File: seedvault-backup-parser.spec (hunk 1 above)
Comment (Rudd-O) [reply seedvault-backup-parser.spec:12]: I will fix this.  Thanks for the feedback.

**Comment by Rudd-O** [comment]
Will get fixed, but I have to open a different PR because Github forces me to.

**Pull request closed without merging**